samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -90,6 +90,16 @@ static void stream_recv_cb(struct bt_bap_stream *stream,
 {
 	static uint32_t recv_cnt;
 
+	if (info->flags & BT_ISO_FLAGS_ERROR) {
+		printk("ISO receive error\n");
+		return;
+	}
+
+	if (info->flags & BT_ISO_FLAGS_LOST) {
+		printk("ISO receive lost\n");
+		return;
+	}
+
```

Comment:
Looking at the flag definitions, we also have BT_ISO_FLAGS_TS, but it's alright to use the data even though we only miss the timestamp ?

Reply:
Yes, the timestamp is optional :)